manager/src/main/resources/application.properties

```diff
@@ -133,6 +135,15 @@ rhoas.max_retries=5
 rhoas.jitter=0.1
 rhoas.backoff=PT0.5S
 
+#DNS OPENSHIFT
+event-bridge.dns.subdomain=${EVENT_BRIDGE_DNS_SUBDOMAIN:.smartevents.bf2.dev}
```

Comment:
@r00ta I'm not sure about providing defaults here. I think it's safer for us to fail to boot the Fleet Manager if the property value is missing rather than getting to the point where we're trying to create the DNS entry in the incorrect zone for the environment that the Fleet Manager is running in.